precommit/src/main/shell/plugins.d/rstcheck.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# SHELLDOC-IGNORE
+
+add_test_type rstcheck
+
+RSTCHECK_TIMER=0
+
+RSTCHECK=${RSTCHECK:-$(command -v rstcheck 2>/dev/null)}
+
+function rstcheck_usage
+{
+  yetus_add_option "--rstcheck=<file>" "path to rstcheck executable"
+}
+
+function rstcheck_parse_args
+{
+  local i
+
+  for i in "$@"; do
+    case ${i} in
+      --rstcheck=*)
```

Comment:
This should be `--rstcheck-cmd`, so as to be consistent with our other plugins that permit specifying the executable path?

Reply:
precommit is _wildly_ inconsistent here.  Some use `--foo-cmd=` and some use `--foo=`.  There should really be a follow-on JIRA to make these consistent.